mne/report.py

```diff
@@ -1747,34 +1729,6 @@ def _render_toc(self, verbose=None):
         self.html.insert(0, html_header)  # Insert header at position 0
         self.html.insert(1, html_toc)  # insert TOC
 
-    def _render_array(self, array, global_id=None, cmap='gray',
```

Comment:
This is all crufty code that isn't used anymore (which is good, because it's wrong when the axes need to be flipped)